modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -720,6 +720,57 @@ definition {
 			actionName = "Update Action Name");
 	}
 
+	@description = "LPS-156347 - Verify that the expression works with Webhooks"
+	@priority = "3"
+	test CanUseExpressionWithWebhook {
+		task ("Given an object with a field is created") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 185",
+				objectName = "CustomObject185",
+				pluralLabelName = "Custom Objects 185");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customObjectField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject185");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject185");
+		}
+
+		task ("When an action is created with an Expression using Webhooks") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 185");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionName = "Action Name",
+				active = "true",
+				inputExpression = "customObjectField == 'Entry Test'",
+				thenAction = "Webhook",
+				urlAction = "http://localhost:8080",
+				whenAction = "On After Add");
+		}
```

Comment:
Hey @chaowang-123, would this on CI? We may need to add the property to skip portal instances for this test. Can you verify?

If it works as is, you can forward this to BChan.